opentelekomcloud/services/elb/v3/resource_opentelekomcloud_lb_loadbalancer_v3.go

```diff
@@ -89,6 +89,14 @@ func ResourceLoadBalancerV3() *schema.Resource {
 				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"public_ip_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"public_ip_address": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
```

Comment:
**Optional:** Those fields can be just `id` and `address`, as the parent field is named `public_ip`